src/hotspot/share/jfr/jni/jfrJavaSupport.cpp

```diff
@@ -37,6 +37,7 @@
 #include "oops/klass.inline.hpp"
 #include "oops/oop.inline.hpp"
 #include "oops/objArrayKlass.hpp"
+#include "oops/objArrayOop.hpp"
 #include "oops/objArrayOop.inline.hpp"
```

Comment:
We usually don't explicitly include the .hpp file when the .inline.hpp file is already included.

Reply:
(I also see that the lines many a few of the include  lines are not sorted)